oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -223,6 +223,8 @@ configuration:
 #    hostAndPorts: ${consul.address}
 #    # Sync period in seconds. Defaults to 60 seconds.
 #    period: 1
+#    # Consul aclToken
+#    #aclToken: ${consul.aclToken}
```

Comment:
There is another application.yml in dist-material fold, please sync the change there too.